### ISSUE TITLE
I look forward to implementing your great work in Dog Matrix - AMO

### ISSUE THREAD

**Issue by LauOtero** [moggieuk/Happy-Hare#110]
![dog_matrix](https://github.com/moggieuk/Happy-Hare/assets/3716039/4ef02ad1-14d1-47b6-8815-f4f516adb19c)

Great job on the new version, it makes my life easier for the new project I am developing.

It looks like it contemplates a design similar to Bambu Lab's AMS, may my design be based on that system.

Dog Matrix - AMO
![dog_matrix2](https://github.com/moggieuk/Happy-Hare/assets/3716039/ee72c7e8-f72b-4385-a822-c4f49f5ad66c)

My intention is to have 2 versions, normal and XL for 2kg filament spools.
It would be great to implement a solution for the Knomi V1 and V2 touch.

Thank you for your great work!!! 👏🫶

**Comment by erazor666** [comment]
Looks cool! Is there any public info on this unit yet? Always looking for my next project 😎

**Comment by moggieuk** [comment]
Very interesting.  So I would describe this as a "type-B" MMU.  See: https://github.com/moggieuk/Happy-Hare/blob/main/doc/conceptual_mmu.md

Type-B support is almost done.  In fact I was doing some work today on it...

**Comment by erazor666** [comment]
@LauOtero this looks really cool and like a very worthwhile project, how far along is it? Any github?

**Comment by LauOtero** [comment]
> @LauOteroEsto se ve realmente genial y es un proyecto que vale la pena, ¿qué tan avanzado está? ¿Algún github?

The design of certain components still needs to be finished in order to start testing and verify that everything works correctly.

One of the mods I'm designing is a dehumidifier, it complicates the design a bit trying to make it as airtight as possible.

There is private github but it is in Spanish, STEPS and BOM of some component, little documentation. I also manage with a partner a telegram channel dedicated to klipper in Spanish with a specific topic with the progress of the project.

Basically I'm doing the project by myself, when it's more advanced I hope to have more collaborators.

**Comment by LauOtero** [comment]
Exactly, it is a type B MMU, in fact it is based on Bambu Lab's AMS.

It is designed to not perform calibrations, here (Spain) do not like much calibration with the encoder, I try to avoid any kind of calibration, I want it to be plug & play.

I adapt to what you program, I need to study the operation of the MMU in depth but I do not have the hours in the day to do it all.

Thanks for the great work you do.

> Very interesting. So I would describe this as a "type-B" MMU. See: https://github.com/moggieuk/Happy-Hare/blob/main/doc/conceptual_mmu.md
> 
> Type-B support is almost done. In fact I was doing some work today on it...


Exactly, it is a type B MMU, in fact it is based on Bambu Lab's AMS.

It is designed to not perform calibrations, here (Spain) do not like much calibration with the encoder, I try to avoid any kind of calibration, I want it to be plug & play.

I adapt to what you program, I need to study the operation of the MMU in depth but I do not have the hours in the day to do it all.

Thanks for the great work you do.
